src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -90,6 +90,25 @@ private void SetMembraneTooltips(MembraneType referenceMembrane)
             var tooltip = GetSelectionTooltip(membraneType.InternalName, "membraneSelection");
             tooltip?.WriteMembraneModifierList(referenceMembrane, membraneType);
         }
+
+        // Osmoregulation cost is based on the membrane, so update the osmoregulation tooltips
+        // after updating the membrane
+        SetOsmoregulationTooltips();
+    }
+
+    private void SetOsmoregulationTooltips()
```

Comment:
```suggestion
    private void UpdateOsmoregulationTooltips()
```

I think this kind of name would be more consistent with other parts of the editor.